tests/functional/error-bad-types/expected-stdout

```diff
@@ -1,5 +1,12 @@
-Exporting to in.c-main.escaping
-DB query "SELECT CAST("decimal" AS DECIMAL(15,5)), "character", "integer", "date" FROM "types"" failed: : Error "odbc_prepare(): SQL error: SQL compilation error: Object '"types"' does not exist or not authorized., SQL state S0002 in SQLPrepare" while executing query "SELECT * FROM (SELECT CAST("decimal" AS DECIMAL(15,5)), "character", "integer", "date" FROM "types") LIMIT 0;". Retrying... [1x]
-DB query "SELECT CAST("decimal" AS DECIMAL(15,5)), "character", "integer", "date" FROM "types"" failed: : Error "odbc_prepare(): SQL error: SQL compilation error: Object '"types"' does not exist or not authorized., SQL state S0002 in SQLPrepare" while executing query "SELECT * FROM (SELECT CAST("decimal" AS DECIMAL(15,5)), "character", "integer", "date" FROM "types") LIMIT 0;". Retrying... [2x]
-DB query "SELECT CAST("decimal" AS DECIMAL(15,5)), "character", "integer", "date" FROM "types"" failed: : Error "odbc_prepare(): SQL error: SQL compilation error: Object '"types"' does not exist or not authorized., SQL state S0002 in SQLPrepare" while executing query "SELECT * FROM (SELECT CAST("decimal" AS DECIMAL(15,5)), "character", "integer", "date" FROM "types") LIMIT 0;". Retrying... [3x]
-DB query "SELECT CAST("decimal" AS DECIMAL(15,5)), "character", "integer", "date" FROM "types"" failed: : Error "odbc_prepare(): SQL error: SQL compilation error: Object '"types"' does not exist or not authorized., SQL state S0002 in SQLPrepare" while executing query "SELECT * FROM (SELECT CAST("decimal" AS DECIMAL(15,5)), "character", "integer", "date" FROM "types") LIMIT 0;". Retrying... [4x]
+Creating ODBC connection to "Driver=SnowflakeDSIIDriver;Server=kebooladev.snowflakecomputing.com;Port=443;Tracing=0;Database="%s";Schema="%s"".
+Use schema "%s" in database "%s"
+Exporting "sales" to "in.c-main.escaping".
+Creating ODBC connection to "Driver=SnowflakeDSIIDriver;Server=kebooladev.snowflakecomputing.com;Port=443;Tracing=0;Database="%s";Schema="%s"".
+SQL compilation error: Object '"types"' does not exist or not authorized. S0002. Retrying... [1x]
+Creating ODBC connection to "Driver=SnowflakeDSIIDriver;Server=kebooladev.snowflakecomputing.com;Port=443;Tracing=0;Database="%s";Schema="%s"".
+SQL compilation error: Object '"types"' does not exist or not authorized. S0002. Retrying... [2x]
```

Comment:
Na SQL compilation error by nemalo byt retry, ale vidim, ze to tak uz bolo. Upravim vramci svojho tasku.